src/components/screens/NewsItem.js

```diff
@@ -14,6 +15,16 @@ import { ImagesCarousel } from '../ImagesCarousel';
 import { momentFormat, openLink, trimNewLines } from '../../helpers';
 import { BoldText, RegularText } from '../Text';
 
+// necessary hacky way of implementing iframe in webview with correct zoom level
+// thx to: https://stackoverflow.com/a/55780430
+const INJECTED_JAVASCRIPT_FOR_IFRAME_WEBVIEW = `
```

Comment:
Is this a sort of importing ?

Reply:
no, it is a constant definition for using it as the prop `injectedJavaScript` inside of the WebView